docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc

```diff
@@ -14,3 +32,76 @@ The enhanced link daemon collects xref:operation:topology/enlinkd/introduction.a
 | enlinkd.log
 | Logs relating to enlinkd server events.
 |===
+
+
+== Enlinkd configuration
+
+If multiple protocols are enabled, links will be discovered for each enabled discovery protocol.
+The topology web UI visualizes links for each discovery protocol.
+For example, if you start CDP and LLDP discovery, the web UI visualizes a CDP link and an LLDP link.
+
+Restart {page-component-title} or just the enlinkd daemon to apply configuration changes.
```

Comment:
The 
enlinkd configuration file over table says restart is not required, but this text indicates the opposite.

Reply:
This is a wording issue that plagues all the daemon files. A restart of the full horizon/Meridian server isn't required to apply changes, just a reload of the daemon.

Reply:
We can consider rephrasing if we go through the section again. It's fine for now.